src/layman/common/consts.py

```diff
@@ -0,0 +1,6 @@
+REQUEST_METHOD_DELETE = 'delete'
```

Comment:
It's little bit strange we are mixing here HTTP requests and WFS requests as they are from different network levels.

I suggest to use one set of variables for HTTP requests and another set for locks (HTTP request names + WFST).

Reply:
Move it to common/__init__.py